xml/storage_multipath.xml

```diff
@@ -4643,6 +4643,32 @@ Give root password for maintenance
      </para>
     </listitem>
    </itemizedlist>
+   <important>
+    <para>
+     At this stage, you are working in a <command>dracut</command> emergency
+     shell from the initrd environment. All modification resulting from the following
+     procedures will be lost at the next reboot. To make them persistent, you
+     need to either:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       Mount the system root (<filename>/</filename>) file system and
+       <command>chroot</command> to it from the <command>dracut</command> emergency shell. You may
+       need to mount the <filename>/proc</filename>, <filename>/sys</filename>,
+       and <filename>/dev</filename> partitions as well if dracut needs access
+       to them.
```

Comment:
What about `/run` ? Also, more often than not, *after* chroot, it may be necessary to mount additional file systems (subvolumes), like this:

    mount -a -t btrfs